swfiles/@spinw/genlattice.m

```diff
@@ -51,7 +51,7 @@
 % : `[a, b, c]` lattice parameters in units defined in [spinw.unit] (with \\ang
 %   being the default), dimensions are $[1\times 3]$.
 % 
-% `spgr`
+% `spgr` or 'sym'
```

Comment:
Good thinking I will add the test for both supplied in the same call. Yes I think given you can supply your own symmetry operations and not the spacegroup number/symbol perhaps the newer `sym` would be a preferable name but `spgr` is in all the tutorials (and probably user's code given `sym` was undocumented) so perhaps it's not worth changing? I can add a deprecation warning for `sym`?

Reply:
`sym` is more widely used and descriptive so we've decided to deprecate `spgr`

Reply:
Currently if both `sym` and `spgr` are provided then `spgr` will be used (`sym` is overwritten by `spgr`) - if we are going to deprecate `spgr` then I think we should have the opposite behaviour:  `sym` should be used over `spgr` if both are provided - is that OK?

Reply:
Alternatively we could error if both are provided?

Reply:
Hmm, yeh maybe an error is better as it probably is a typo in the user script/call to `genlattice` - the same could be said for `angle` and `angled` as well

Reply:
Yeah that is true. There is an instance of that in `genmagstr` as well (`phi` and `phid`), although I'm hesitant to try and cover all possible instances of argument clashes (in the long long term maybe Python can help us with argument validation...). I guess just do `sym` and `spgr` for now since we're messing with them anyway and leave others for another time?

Reply:
OK so to confirm - throw an error if both `sym` and `spgr` provided?